emails/models.py

```diff
@@ -390,6 +393,20 @@ def joined_before_premium_release(self):
         date_created = self.user.date_joined
         return date_created < datetime.fromisoformat("2021-10-22 17:00:00+00:00")
 
+    @property
+    def date_phone_registered(self) -> datetime | None:
+        if not settings.PHONES_ENABLED:
+            return None
+
+        try:
+            real_phone = RealPhone.objects.get(user=self.user)
+            relay_number = RelayNumber.objects.get(user=self.user)
+        except RealPhone.DoesNotExist:
+            return None
+        except RelayNumber.DoesNotExist:
+            return real_phone.verified_date
+        return relay_number.created_at or real_phone.verified_date
```

Comment:
Clever, I wondered how you were going to do this without populating `created_at`